examples/main/README.md

```diff
@@ -181,7 +181,7 @@ Example usage: `--temp 0.5`
 
 ### Repeat Penalty
 
--   `--repeat-penalty N`: Control the repetition of token sequences in the generated text (default: 1.1).
+-   `--repeat-penalty N`: Control the repetition of token sequences in the generated text (default: 1.1, 1.0 = disabled).
```

Comment:
The default was changed to `1.0` (in <https://github.com/ggerganov/llama.cpp/pull/6127>), so maybe this should be updated too?